### DISCUSSION TITLE
UCB CS285: Deep Reinforcement Learning - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#122] [Announcements]
# UCB CS285: Deep Reinforcement Learning - CS自学指南

CS自学指南

https://csdiy.wiki/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0/CS285/?h=reinfor

**Comment by SaltAdamW** [comment]
终于找到佬没学的课程了XD

**Comment by GlacierGao** [comment]
wow，大佬更新了。非常期待强化学习相关的课程。我是交通工程专业的，这学期学了信号灯配时（交通管控），好像最近有很多成果用上了强化学习方法，但是课上一点也没有介绍到~谢谢！

**Comment by ArchyZheng** [comment]
有人一起上这门课吗 cs285 2023 fall. 可以一起讨论一下作业吗, 因为这门课没有提供作业的答案.

**Reply by zichi-jin** [reply]
同23fall，请问可以一起讨论下作业吗

**Comment by bowieshi** [comment]
正在做2023fall的作业，但网上还没有开源的标准答案也不知道对不对，我自己写的放在https://github.com/bowieshi/UCB_CS285了，有兴趣一起讨论的同学可以email我

**Comment by weqeweqewe** [comment]
强化学期可以看看 西湖大学的课 很不错

**Comment by hanshublyh** [comment]
关于推导证明的作业有佬做嘛？

**Reply by hanshublyh** [reply]
23fall的

**Comment by rinevard** [comment]
建了一个qq群：
642769874
我目前也正在学习CS285

**Comment by MTDickens** [comment]
新手如果觉得这个课程太难的话，可以选择先看 David Silver 的（[链接](https://www.davidsilver.uk/teaching)）。虽然 David 的课已经是 2015 年的老课了，但是 RL 的基本思想还是没变的。

**Comment by Benson722** [comment]
本人正在学习该课程，欢迎来我的repo对答案！共同学习，共同进步:)

https://github.com/Benson722/homework_fall2023

**Comment by ccmoony** [comment]
这是我的2023 fall作业的代码实现，欢迎交流！
https://github.com/ccmoony/UCB-CS285.git

**Comment by songgaogy** [comment]
UCB的RL课确实不错。上过斯坦福的cs234 RL，感觉远不如这门cs285。Levine教授深入浅出，intuition讲的十分透彻。当然由于课程内容原因，对初学不是那么友好。谢谢大佬指明方向～

**Reply by Andy3117006664** [reply]
cs234 RL作业设计的怎么样

**Reply by Andy3117006664** [reply]
这门cs285的slides我感觉比较好，CS234的slides偏向David Silver的风格

**Reply by songgaogy** [reply]
cs234 RL作业比较理论，对于收敛性与一些细节（比如对于bellman 算子性质）要求多。虽然这在研究中是必须的，但是感觉对于学习RL还是有些“不令人舒服”（个人想法哈）

**Comment by WILLREALME** [comment]
西湖大学赵老师的课看完之后，再来看这个吧

**Comment by Vitoom** [comment]
学习这门课程的预备课程：https://www.zhihu.com/question/61171437/answer/876455948?share_code=544RFUBwaQPc&utm_psn=1915072972378600824

**Comment by Perian-Yan** [comment]
刚开始学了两周，自己在尝试做作业https://github.com/Perian-Yan/berkeley_drl_homework_fall2023
欢迎一起讨论

**Comment by habaohaba** [comment]
record里面lec 3没有了吗

**Comment by yulongshan** [comment]
正在学2026年的课程，有队友一起吗

**Reply by habaohaba** [reply]
打算是做这个学期proj吗？然后看往年的recording？👀

**Reply by yulongshan** [reply]
看的26年的ppt，只看ppt和做作业，目前lecture看到第10节了

**Reply by soundstarrain** [reply]
我打算在近期开始学习这门课程。

**Reply by yulongshan** [reply]
那期待有机会可以一起交流下

**Reply by 2023lyan** [reply]
我也打算开始学习这个课程，有机会可以一起交流

**Reply by yulongshan** [reply]
期待

**Reply by 9Skie** [reply]
俺也有打算

**Reply by EnzoDing-rgb** [reply]
搞个交流群吧

**Reply by jiucai233** [reply]
可以交流啊，我现在做到offline RL了

**Reply by YitaoXU** [reply]
有无同学一起交流学习一下

**Reply by 9Skie** [reply]
我去看Stanford cs224R了，我自己还是比较喜欢有视频的
